ui/apps/platform/cypress/constants/PoliciesPage.js

```diff
@@ -100,6 +100,8 @@ export const selectors = {
         renameInput: 'input#policy-rename',
         overwriteRadioLabel:
             'label[for="policy-overwrite-radio-1"]:contains("Overwrite existing policy")',
+        errorAlertTitle:
+            '.pf-c-modal-box__body .pf-m-danger .pf-c-alert__title:contains("Errors trying to import policies")',
```

Comment:
Haha, I read as if the errors are trying to import the policies.

Especially for global audience, because other languages do not have **ing** form, meaning usually becomes clearer when we write verb without it.

Reply:
If this means that the import can't proceed because the policies that you are trying to import have errors:

Policy errors causing import failure

?